manager/src/main/resources/application.properties

```diff
@@ -21,8 +21,8 @@ quarkus.flyway.ignore-missing-migrations=true
 quarkus.flyway.create-schemas=true
 
 # SSO
-quarkus.oidc.auth-server-url=${EVENT_BRIDGE_FM_SSO_URL}
-quarkus.oidc.client-id=${EVENT_BRIDGE_FM_SSO_CLIENT_ID}
+quarkus.oidc.auth-server-url=${EVENT_BRIDGE_SSO_URL}
+quarkus.oidc.client-id=${EVENT_BRIDGE_FM_CLIENT_ID}
```

Comment:
@fjtirado Think this one still needs updating.

Reply:
done